Makefile

```diff
@@ -197,6 +202,7 @@ superuser: ## Create an admin user with password "admin"
 	@$(COMPOSE) up -d mysql
 	@echo "Wait for services to be up..."
 	@$(WAIT_DB)
+	@$(MANAGE) migrate
```

Comment:
I don't think we want to force migrate when creating a user. Can you remove this change?

Reply:
Yes, sure.
Done